Source/WebCore/platform/LengthPoint.h

```diff
@@ -32,36 +32,27 @@ namespace WebCore {
 struct BlendingContext;
 
 struct LengthPoint {
-public:
     LengthPoint()
     {
     }
-    
+
```

Comment:
Could we eliminate this constructor?

Reply:
We need to keep this default constructor since other instances of LengthPoint are already using it. The compiler will not generate this constructor automatically for us if we also provide the constructor LengthPoint(x, y) unless we keep this or we write: `LengthPoint() = default; `

Reply:
`LengthPoint() = default;` is more elegant

Reply:
Done.